docs/source/getting_started.md

```diff
@@ -27,288 +27,262 @@ kernelspec:
 ```{code-cell}
-p = thermal_block_problem(num_blocks=(3, 2))
+fom_tb.parameters
 ```
 
-We want to discretize this problem using the finite element method.
-We could do this by hand, creating a {{ Grid }}, instatiating
-{class}`~pymor.discretizers.builtin.cg.DiffusionOperatorP1` finite element diffusion
-operators for each subblock of the domain, forming a {{ LincombOperator }}
-to represent the affine decomposition, instantiating a
-{class}`~pymor.discretizers.builtin.cg.L2ProductFunctionalP1` as right hand side, and
-putting it all together into a {{ StationaryModel }}. However, since
-{meth}`~pymor.analyticalproblems.thermalblock.thermal_block_problem` returns
-a {class}`~pymor.analyticalproblems.elliptic.StationaryProblem`, we can use
-a predifined *discretizer* to do the work for us. In this case, we use
-{func}`~pymor.discretizers.builtin.cg.discretize_stationary_cg`:
+Let us show the solution for particular parameter values:
 
 ```{code-cell}
-fom, fom_data = discretize_stationary_cg(p, diameter=1./50.)
+mu = [0.1, 0.2, 0.5, 1]
+U = fom_tb.solve(mu)
+fom_tb.visualize(U)
 ```
 
-`fom` is the {{ StationaryModel }} which has been created for us,
-whereas `fom_data` contains some additional data, in particular the {{ Grid }}
-and the {{ BoundaryInfo }} which have been created during discretization. We
-can have a look at the grid,
+To construct a reduced-order model using the reduced basis method,
+we first need to create a {mod}`reductor <pymor.reductors>` which projects
+the model onto a given reduced space:
 
 ```{code-cell}
-print(fom_data['grid'])
+from pymor.parameters.functionals import ExpressionParameterFunctional
+from pymor.reductors.coercive import CoerciveRBReductor
+
+reductor = CoerciveRBReductor(
+    fom_tb,
+    product=fom_tb.h1_0_semi_product,
+    coercivity_estimator=ExpressionParameterFunctional('min(diffusion)',
+                                                       fom_tb.parameters)
+)
 ```
 
-and, as always, we can display its class documentation using
-`help(fom_data['grid'])`.
+We use here {class}`~pymor.reductors.coercive.CoerciveRBReductor`, which
+also assembles an a posterior error estimator for the error w.r.t. the
+$H^1_0$-seminorm.
+For that estimator, an parameter-dependent lower bound for the coercivity constant
+of the problem is provided as an {class}`~pymor.parameters.functionals.ExpressionParameterFunctional`.
 
-Let's solve the thermal block problem and visualize the solution:
+Using this reductor and its error estimator, we can build a reduced space with corrsponding
+reduced-order model using a {func}`~pymor.algorithms.greedy.weak_greedy` algorithms:
 
 ```{code-cell}
-U = fom.solve([1.0, 0.1, 0.3, 0.1, 0.2, 1.0])
-fom.visualize(U, title='Solution')
+from pymor.algorithms.greedy import rb_greedy
+
+parameter_space = fom_tb.parameters.space(0.1, 1)
+greedy_data = rb_greedy(fom_tb, reductor, parameter_space.sample_randomly(1000),
+                        rtol=1e-2)
+rom_tb = greedy_data['rom']
 ```
```

Comment:
Text mentions `weak_greedy`, but the code uses `rb_greedy`. @sdrave Is this an issue?

Reply:
I rephrased it a bit ..